gwcli/storage.py

```diff
@@ -750,6 +803,9 @@ def ui_command_resize(self, size=None):
 
         self.resize(size)
 
+    def ui_command_reconfigure(self, attribute, value):
+        self.reconfigure(attribute, value)
```

Comment:
... still need to add the documentation (derived from the ```ui_command_reconfigure``` above)

Reply:
done